package.json

```diff
@@ -24,15 +24,22 @@
   },
   "homepage": "https://github.com/SvenKolaric/learnQuery#readme",
   "dependencies": {
+    "editorconfig": "^0.15.3",
     "eslint-config-airbnb-base": "^14.1.0",
     "typescript": "^3.9.3"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
```

Comment:
actually ... i would add a pre-push hook 
since some things could be "WIP" and you just want to commit them - but it's on you, what suits you best. 
this is a non blocker.

Reply:
I will follow your advice, will change the hook to pre-push and merge to master